### PR TITLE
fix: add idPrefix prop to EzLineChart

### PR DESCRIPTION
**Pull request by danidewitt**
## What did we change?
- Add `idPrefix` prop to `EzLineChart`

## Why are we doing this?
- SSR id issues

## Screenshot(s) / Gif(s):
Screen Shot 2023-05-31 at 3 41 08 PM

## Checklist

- [x] Provide test coverage for the changes made
- [x] Create a changeset (`yarn changeset`) with a summary of the changes
 
[Contributing guidelines](https://ezcater.github.io/recipe/guides/contributing/)

### CODE REVIEW

**Review by noranda — APPROVED** [review]

**Review by pgrippi — APPROVED** [review]
🎱 thoughts on using `id` instead of `customId`? It is eventually used as part of the id element on the actual DOM nodes

**Comment by agrobbin** [comment]
I like `id` or perhaps `containerId`, to match Victory's interface.

**Comment by agrobbin** [comment]
I think I would still prefer `containerId` based on how it's passed down into the Victory component, but I don't have a really strong opinion here.

**Review by onocy — DISMISSED** [review]

File: packages/recipe/src/components/EzLineChart/__tests__/EzLineChart.test.tsx [hunk 1]
```diff
@@ -21,25 +21,31 @@ const DEPENDENT_AXIS_LABEL_VALUES = [0, 1000, 2000, 3000, 4000, 5000];
...
+        title={TITLE}
       />
     );
 
-    expect(getByLabelText(LABEL)).toBeDefined();
+    const chartTitle = getByLabelText(TITLE);
+    const titleId = chartTitle.querySelector(`#${ID_PREFIX}-title`);
+
+    expect(chartTitle).toBeDefined();
+    expect(titleId).toBeDefined();
```
Comment (pgrippi) [inline packages/recipe/src/components/EzLineChart/__tests__/EzLineChart.test.tsx:48]: these should both be [`.toBeInTheDocument()`](https://github.com/testing-library/jest-dom#tobeinthedocument)

**Review by pgrippi — APPROVED** [review]

**Review by noranda — APPROVED** [review]

**Review by pgrippi — APPROVED** [review]

**Review by noranda — APPROVED** [review]

**Pull request merged**